<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;</string>
    </property>
    <property name="text">
     <string>Nombre de usuario:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;</string>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textoUsuario">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textoPassword">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="botonIngresar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#botonIngresar {
	background-color: rgb(32, 74, 135);
	color: white;
}

#botonIngresar:hover {
	background-color: rgb(78, 154, 6);
}</string>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;
color: rgb(46, 52, 54);</string>
    </property>
    <property name="text">
     <string>¿No dispones de una cuenta? Regístrate dando </string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>691</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 32px;
color: white;</string>
     </property>
     <property name="text">
      <string>INICIO DE SESIÓN</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelUsuarioIncorrecto">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;
color: red;</string>
    </property>
    <property name="text">
     <string>Usuario o contraseña incorrectos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonAbrirFormRegistrar">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#botonIngresar {
	background-color: rgb(32, 74, 135);
	color: white;
}

#botonIngresar:hover {
	background-color: rgb(78, 154, 6);
}</string>
    </property>
    <property name="text">
     <string>Click aquí</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
